packages/date-parser/src/helpers/weekdayIdxFromLuxon.js

```diff
@@ -0,0 +1,6 @@
+const weekdayIdxFromLuxon = (luxon) => {
+  const idx = luxon.weekday;
+  return idx === 7 ? 0 : idx; // our Sunday starts at 0, while Luxon is 7
```

Comment:
```suggestion
  return idx === 7 ? 0 : idx; // our Sunday is 0, while Luxon is 7
```
'starts' sounds like Luxon goes from 7 to 13